src/main/kotlin/com/google/androidstudiopoet/ModuleBlueprintFactory.kt

```diff
@@ -35,6 +35,6 @@ object ModuleBlueprintFactory {
          return AndroidModuleBlueprint(i,
                  configPOJO!!.numActivitiesPerAndroidModule!!.toInt(),
                  configPOJO.numActivitiesPerAndroidModule!!.toInt(),
-                 configPOJO.numActivitiesPerAndroidModule.toInt(), projectRoot, i == 0, dependencies, configPOJO.productFlavors)
+                 configPOJO.numActivitiesPerAndroidModule.toInt(), projectRoot, i == 0, configPOJO.kotlinPackageCount!!.toInt() > 0, dependencies, configPOJO.productFlavors)
```

Comment:
If I'm not mistaking then you already added `useKotlin` flag in `ConfigPOJO` can you please use it here?

Reply:
Thanks for this comment, this is exactly why I added it on ConfigPOJO (I was using the same condition in multiple places)